syntax = "proto3";

package SvrCmd;
enum EnumSvrCmd
{
	CMD_BEGIN_FALG = 0;
	CMD_GET_GAMESVR_INSTANCE = 4000;
	CMD_COMMON_QUERY = 4001;
	CMD_SUBMIT_GAME_RESULT = 4002;
	CMD_USERLOG_RECORD = 4003;
	CMD_GAMELOG_RECORD = 4004; 
	CMD_APPLOG_RECORD = 4005; 
	CMD_ADD_PLAYER_CHIPS = 4006;
	CMD_APPLY_BUYIN = 4007;
	CMD_USER_JOIN_GAME = 4008;
	CMD_USER_LEAVE_GAME = 4009;
	CMD_USER_ENTER_ROOM = 4010;
	CMD_USER_QUIT_ROOM = 4011;
	CMD_GAMESVR_START = 4012;
	CMD_OPEN_GAME_ROOM = 4013;
	CMD_GAME_ROOM_EMPTY = 4014;
	CMD_GAMESVR_ALIVE = 4015; 
	CMD_USER_CHIPS_LOG = 4016;
	CMD_HALL_START_UP = 4017;
	CMD_FLUSH_USER_ONLINE = 4018;
	CMD_TELL_USER_OFFLINE = 4019;
	CMD_FLUSH_USER_IN_GAME = 4020;
	CMD_DISPATCH_USER_MSG = 4021;
	CMD_HALL_LOGIN_RECORD = 4022;
	CMD_GAME_LOGIN_RECORD = 4023;
	CMD_CHECK_USER_TOKEN = 4024;
	CMD_KICK_OUT_USER = 4025;
	CMD_GET_INGAME_COUNT = 4026;
	CMD_CHECK_USER_LOGIN = 4027;
	CMD_FORBIDDEN_USER = 4028;
	CMD_FLUSH_USER_GAMESESSION = 4029;
	CMD_TELL_USER_LOGOUT_GAME = 4030;
	CMD_GAME_BUY_CHIPS_IN = 4031;
	CMD_GAME_OVER_ACCOUNT = 4032;
	CMD_GAME_LEAVE_ACCOUNT = 4033;
    CMD_USER_BOX_TIME_RECOVER = 4034;
    CMD_USER_BOX_TIME_PAUSE = 4035;
    CMD_GAME_START_BET = 4036;
    CMD_GAME_ADD_CARRY = 4037;
    CMD_MATCH_OVER_ACCOUNT = 4038;
    CMD_SNGLOG_RECORD = 4039;
    CMD_SNG_GAMELOG_RECORD = 4040;
	CMD_BANKRUPT_LOG = 4041;
	CMD_USER_LOGIN_DCENTER = 4042;
	CMD_USER_LOGIN_GAMESVR = 4043;
	CMD_NOTIFY_OPEN = 4044;
	CMD_QUERY_ROOMMATCHER = 4045;
	CMD_ROOMMATCHER_STATUS = 4046;
	CMD_NOTIFY_TOKEN_LOGIN = 4047;
}

